Gemfile.lock

```diff
@@ -91,4 +91,4 @@ DEPENDENCIES
   unicorn
 
 BUNDLED WITH
-   1.10.4
+   1.10.3
```

Comment:
Why did bundler version go down?
Also is there a way to remove this? Bundler version doesn't really matter so much as the versions of the gems it's bundling :package: :stuck_out_tongue_winking_eye: